application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -143,7 +148,10 @@ export default {
       }
     },
     escapedName() {
-      const name = escapeHtml(this.name); 
+      let name = escapeHtml(this.name);
+      if(this.isExternal) {
+        name = name.concat(' ').concat('(').concat(this.statusMap.external).concat(')');
```

Comment:
```suggestion
        name = `${name} (${this.statusMap.external})`;
```
Can you try to use this syntax ? It is the one advised by boubaker, instead of using concat. 
The exact syntax must be checked and tests

More indo : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals